samples/license_plate_recognition/overlay.py

```diff
@@ -0,0 +1,44 @@
+"""Draw func adds license plate"""
+from savant_rs.draw_spec import LabelDraw, ObjectDraw
+
+from savant.deepstream.drawfunc import NvDsDrawFunc
+from savant.meta.object import ObjectMeta
+
+
+class Overlay(NvDsDrawFunc):
+    def override_draw_spec(
+        self, object_meta: ObjectMeta, draw_spec: ObjectDraw
+    ) -> ObjectDraw:
+        """Override draw spec for objects with label 'lpd' (license plate detection).
+        Add classifier attributes labels to the object visualisation.
+        """
+        if object_meta.label == 'lpd':
+            new_label_format = draw_spec.label.format
+
+            # one attribute per line
+            # if there's no specific attribute for the object on this frame
+            # reserve a line for it anyway
+            # so that the object's labels don't jump up and down
```

Comment:
The comment doesn't seem to be fitting for this sample.

Reply:
fixed